electron-app/src/server/websites/e621/e621.service.ts

```diff
@@ -175,22 +175,23 @@ export class e621 extends Website {
 
         let ifYouWantToCreateNotice = true;
         function tagIsInvalid(tag: string) {
-          const createTagNotice = `If you want to create a new tag, make a post with it, and then go to the https://e621.net/tags?search[name]=${tag}, press edit and select tag category`;
+          const wikiLink = `See https://e621.net/wiki_pages/show_or_new?title=${tag}.`
+          const createTagNotice = ` ${wikiLink} If you want to create a new tag, make a post with it, and then go to the https://e621.net/tags?search[name]=${tag}, press edit and select tag category`;
```

Comment:
Moved wiki link to the section that shows only once. I guess that should fix most issues in case of big invalid tag amount